idunn/datasources/wikidata.py

```diff
@@ -0,0 +1,98 @@
+from elasticsearch import (
+    Elasticsearch,
+    ConnectionError,
+    NotFoundError,
+    ElasticsearchException,
+)
+import logging
+from idunn import settings
+from idunn.utils import prometheus
+from idunn.utils.redis import RedisWrapper
+
+
+logger = logging.getLogger(__name__)
+
+
+class WikidataEs:
```

Comment:
This name is consistent with the names of the ES indices (`wikidata_en`,  `wikidata_fr`, etc.)   
But it's still a bit confusing, as the data is actually about Wikipedia articles.  
Maybe a comment would be useful ?

Reply:
I'll add a short description of this and also rename it to `WikiEs`, it doesn't really fix the confusion but at least doesn't make it worst. As I understood these indexes may be totally unrelated to Wikidata as there was just a confusion to what Wikidata actually was when it was built?